sklearn/covariance/graph_lasso_.py

```diff
@@ -191,6 +191,7 @@ def graph_lasso(emp_cov, alpha, cov_init=None, mode='cd', tol=1e-4,
     else:
         errors = dict(invalid='raise')
     try:
+        d_gap = np.inf
```

Comment:
Alright, I did not make the connection. Still please add an inline comment such as:

```python
try:
    # be robust to the max_iter=0 edge case, see:
    # https://github.com/scikit-learn/scikit-learn/issues/4134
    d_gap = np.inf
    ...
```